### PR TITLE
Fund New Account (Initial Impl)

### PR DESCRIPTION
**Pull request by Jon-edge**
image

image

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206208625333090

### CODE REVIEW

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/modals/FundAccountModal.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        onPress={handleReceive}
+        icon={
+          <View style={style.iconContainer}>
+            <AntDesignIcon name="wallet" {...iconProps} />
+            <AntDesignIcon name="arrowright" {...iconProps} />
+          </View>
+        }
+      />
+    </ModalUi4>
+  )
```
Comment (paullinator) [inline src/components/modals/FundAccountModal.tsx:107]: This really should have an OK button to close the modal. Looks weird with no button.

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/ui4/PromoCardUi4.tsx [hunk 2]
```diff
@@ -34,7 +35,7 @@ export function PromoCardUi4(props: Props) {
   const dispatch = useDispatch()
 
   const { navigation, promoInfo, onClose } = props
-  const { localeMessages, ctaButton, background } = promoInfo
+  const { localeMessages, ctaButton, background, dismissable } = promoInfo
```
[the remark this answers is not in the document]
Comment (Jon-edge) [reply src/components/ui4/PromoCardUi4.tsx:38]: I think the api is easier to understand for non-engineers on the info server if we just use "dismissable."

The info server already will clean this to default to true, but I can also add a default here

**Review by paullinator — CHANGES_REQUESTED** [review]
Buy/Sell list still needs to block light account users from tapping a payment method

**Review by paullinator — APPROVED** [review]

**Pull request merged**